execution/rerun/rerun.go

```diff
@@ -145,10 +141,9 @@ func addFailedMetadata(res result.Result, args []string, add func(res result.Res
 	}
 }
 
-func WriteLastRunInfo() {
-	contents := getJSON(runInfo)
-	failuresFile := filepath.Join(config.ProjectRoot, dotGauge, infoFileName)
-	dotGaugeDir := filepath.Join(config.ProjectRoot, dotGauge)
+func writeFailedMeta(contents string) {
+	failuresFile := filepath.Join(config.ProjectRoot, DotGauge, failedFile)
```

Comment:
There is already a constant for .gauge folder defined in common. We should be using that variable everywhere.